### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Poojasah77**

### CODE REVIEW

**Review by saphalpdyl — CHANGES_REQUESTED** [review]
Make the started comment really small, otherwise good 👍🏼

**Comment by Poojasah77** [comment]
Ok 👍

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-## This is our awesome attendance tracker app
\ No newline at end of file
+# This is our awesome attendance tracker app
+
```
Comment (milangautamDemon) [inline README.md:2]: This should have an underline

Comment (Poojasah77) [reply README.md:2]: But it would look ugly

**Review by saphalpdyl — APPROVED** [review]
👍🏼

**Pull request merged**